<launch>

<!-- MoveBaseSeq node launch and Points Sequence ROS Parameters Loading -->

    <!--Run move_base_seqs node-->
        <node pkg="navigation_goals" type="follow_path.py" name="follow_path" output="screen">
        
        <!-- Load Sequence of Points (Cartesian Coordinates wrt "map" RF, in meters) -->
            <rosparam param="paths_points">[[0,0,0],[0.2,0,0,-0.2,0,0,0.5,0,0],[1,1,0,2,2,0,3,4,0]]</rosparam>
        <!-- Load Sequence of Desired Yaw Angles (no rotations around x and y axes) in degrees-->
            <rosparam param="paths_angles">[[0],[90,0,180], [0,0,0]]</rosparam>

        </node>

</launch>
